core/src/main/java/com/googlecode/psiprobe/tools/AccessorFactory.java

```diff
@@ -19,11 +19,6 @@
 public class AccessorFactory {
 
   /**
-   * Instantiates a new accessor factory.
-   */
-  private AccessorFactory() {}
```

Comment:
Utility classes with only static methods often have private constructors to prevent them from being instantiated. Why remove them?